tox.ini

```diff
@@ -20,10 +20,21 @@ setenv =
 deps = -rrequirements/lint.txt
 commands =
     yamllint -c .yamllint.yaml .
+    # Avoid linting source/api_reference/autosummary
+    # Need bash to able to use wildcards
+    /bin/bash -c 'rstcheck -r docs/source/miscellaneous docs/source/user_guide docs/source/api_reference/*.rst'
```

Comment:
This won't work on Windows. Would `rstcheck -r docs/source/miscellaneous docs/source/user_guide docs/source/api_reference/` work?

Reply:
Will we be testing the lint job on Windows in the future? Unfortunately that won't work because there's a generated directory in `docs/source/api_reference/` that I avoid linting by using the wildcard. Basically, we need to be able to lint everything in `docs/source/` except what's in `docs/source/api_reference/autosummary/` so maybe there's a better way to structure the command without using the wildcard in a situation like this?

Reply:
I tried piping the files from the `find` command into `rstcheck` but it didn't seem to work

Reply:
OK, let's keep it as it is now. Let's try to figure something out later.